internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -125,6 +125,24 @@ func (o *OCMProvider) AddIdentityProvider(clusterSpec *types.ClusterSpec, identi
 	return nil, nil
 }
 
+func (o *OCMProvider) RemoveResources(clusterSpec *types.ClusterSpec, syncSetName string) error {
+	status, err := o.ocmClient.DeleteSyncSet(clusterSpec.InternalID, syncSetName)
+	if err != nil {
+		// the SyncSet may already have been deleted in a previous run and the cluster is
+		// waiting for the resources to be terminated
+		if status == http.StatusNotFound {
+			return nil
+		}
+		return err
+	}
+
+	if status != http.StatusNoContent {
```

Comment:
When is `http.StatusNoContent` returned?

Reply:
On successful delete of the SyncSet. I've tried out the deletion on a cluster and this was what the client returned.